libbeat/docs/command-reference.asciidoc

```diff
@@ -14,7 +14,7 @@
 
 :global-flags: Also see <<global-flags,Global flags>>.
 
-:export-command-short-desc: Exports the configuration or index template to stdout
+:export-command-short-desc: Exports the configuration,index template or a dashboard to stdout
```

Comment:
Sorry, missed this during my first review. You need to add a space after `configuration,`

Reply:
done